runners/apex/src/test/java/org/apache/beam/runners/apex/examples/StreamingWordCountTest.java

```diff
@@ -91,7 +91,7 @@ public void testWindowedWordCount() throws Exception {
     ApexPipelineOptions options = PipelineOptionsFactory.fromArgs(args).withValidation()
         .as(ApexPipelineOptions.class);
     options.setApplicationName("StreamingWordCount");
-    options.setParallelism(1);
+    //options.setParallelism(1);
```

Comment:
Even with the option available, you shouldn't do this, should you? (I didn't read the surrounding code that much - maybe a comment about why you want it to be `1`?)